src/actions/breakpoints/addBreakpoint.js

```diff
@@ -52,6 +53,21 @@ export default async function addBreakpoint(
   const symbols = getSymbols(getState(), source);
   const astLocation = await getASTLocation(source, symbols, newLocation);
 
+  // generatedSource.text, in the case of pretty print or original tab
+  // returns the minified contents, which is why it's used
+  // when creating the "originalText"
+  const generatedSource = getSource(state, generatedLocation.sourceId);
+
+  const text = getTextAtPosition(source.text, location.line, location.column);
+  const originalText =
+    generatedSource && generatedSource.text
+      ? getTextAtPosition(
+          generatedSource.text,
```

Comment:
nit: this could be simplified

```diff
+        getTextAtPosition(
+           generatedSource && generatedSource.text,
+           actualLocation.line,
+           actualLocation.column
+         )
+       ;
```

```